src/tests/ftest/server/multiengine_persocket.yaml

```diff
@@ -76,7 +76,14 @@ server_config:
           class: dcpm
           scm_list: [/dev/pmem1.1]
           scm_mount: /mnt/daos4
-
+  transport_config:
+    allow_insecure: True
+agent_config:
+  transport_config:
+    allow_insecure: True
+dmg:
+  transport_config:
+    allow_insecure: True
```

Comment:
Why do we use `allow_insecure: True`? I thought we should avoid insecure mode unless it's absolutely necessary.

Reply:
maybe something to do with it being to do with a manual test?